decidim-meetings/app/controllers/decidim/meetings/meetings_controller.rb

```diff
@@ -6,9 +6,33 @@ module Meetings
     class MeetingsController < Decidim::Meetings::ApplicationController
       include FilterResource
 
-      helper_method :meetings, :geocoded_meetings, :meeting
+      helper_method :meeting
 
-      def index; end
+      def index
+        results = search.results
+
+        if !params[:filter] && results.length == 0
+          params[:filter] = {
+            date: "past",
+            meetings_warning: true
```

Comment:
This could probably be done by checking if any filter is applied, instead of propagating the parameter around. Looks a bit convoluted to me and we're mixing view logic with controller logic.